packages/select/src/Select/components/TriggerButton/TriggerButton.module.scss

```diff
@@ -21,6 +21,10 @@
   &.placeholder {
     color: rgba($color-purple-800-rgb, 0.7);
   }
+
+  &:disabled {
+    opacity: 0.5;
+  }
```

Comment:
I added this style as a placeholder, it might need refining

Reply:
I will add the proper styling to another card after this has been merged - then I can add a stickersheet example too